src/blog/dark-mode-with-style-dictionary/index.mdx

```diff
@@ -0,0 +1,1352 @@
+    name: 'android',
+    type: 'folder'
+  },{
+    name: 'generateGraphics.js',
+    description: 'Custom action to generate graphics for all platforms',
+    type: 'code',
+  }]
+},{
+  name: 'android',
+  type: 'folder',
+  description: 'iOS demo and generated files from style dictionary'
+},{
+  name: 'assets',
+  type: 'folder',
+  description: 'Source SVG graphics'
+},{
+  name: 'formats',
+  type: 'folder',
+  description: 'Custom format code'
+},{
+  name: 'ios',
+  type: 'folder',
+  description: 'iOS demo and generated files from style dictionary',
+},{
+  name: 'tokens',
+  type: 'folder',
+  description: 'Source token files',
+  items: [{
+    name: 'asset',
+    type: 'folder'
+  },{
+    name: 'color',
+    type: 'folder'
+  },{
+    name: 'size',
+    type: 'folder'
+  },{
+    name: 'component',
+    type: 'folder'
+  }]
+},{
+  name: 'transforms',
+  type: 'folder',
+  description: 'Custom transform code'
+},{
+  name: 'web',
+  type: 'folder',
+  description: 'Web demo and generated web files from style dictionary'
+},{
+  name: 'build.js',
+  type: 'code',
+  description: 'Code that runs Style Dictionary'
+},{
+  name: 'package.json',
+  type: 'code',
+  description: ''
+}]} />
+
+
+### Project setup
+
+1. Grab the repository code: `git clone https://github.com/dbanksdesign/style-dictionary-dark-mode.git`
+1. Get the starter branch: `git fetch && git checkout starter`
+1. Install dependencies: `npm ci`
+1. Run Style Dictionary: `npm run build`
+1. You should see Style Dictionary run and generate some files.
+
+### Running the iOS demo
+
+1. You will need [CocoaPods](https://cocoapods.org/) installed
+1. Go into the **ios/demo** directory with `cd ios/demo`
+1. Run `pod install` to install the style dictionary CocoaOod
+1. Open the Xcode workspace: `open StyleDictionaryDarkModeDemo.xcworkspace`
+1. Click the ▶️ button to build and run the demo
+
+### Running the Android demo
+
+1. Open Android Studio
+1. Click on **Open an existing project**
+1. Navigate to the folder you cloned the repository into and select the **android** folder
+1. Android Studio should run for a bit syncing dependencies and whatnot
+1. Click the play button at the top to build and run the app in an emulator (you will need to have an emulator downloaded already)
+
+### Running the Web demo
+
+1. Go into the **web/demo** directory with `cd web/demo`
+1. Run `npm install` to install dependencies. *This will also create a local dependency on our style dictionary package with a symlink using `npm link`*
+1. Run `npm start` to start [11ty](https://www.11ty.dev/)
+1. Open **http://localhost:8080** in your browser
+
+
+### Graphics in Style Dictionary
+
+This is an interesting topic that deserves its own article. I will briefly touch on it here as it relates to dark mode. Design tokens can encapsulate things more complex than a color or size, they could be a whole graphic or icon too. For example you could create a logo design token that points to an SVG file:
```

Comment:
size, they could be a whole graphic or icon too. > size: they can be a whole graphic or icon.